node/pkg/guardiansigner/generatedsigner.go

```diff
@@ -0,0 +1,53 @@
+package guardiansigner
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+)
+
+type GeneratedSigner struct {
+	privateKey *ecdsa.PrivateKey
+}
+
```

Comment:
I know you have a comment in `guardianSigner.go` about this being for test only, but could you please put something in here as well?

Reply:
I added a comment to the struct. I also got paranoid, and moved the `GenerateSignerWithPrivatekeyUnsafe` function to `generatedsigner.go`.